src/plugins/data_table/data_table.js

```diff
@@ -347,6 +347,11 @@ var LocalDataTable = (function() {
       }
     },
 
+    _columnElements: function(selector) {
+      var selectorString = selector || "";
+      return this.$("table").find("thead tr th" + selectorString);
+    },
```

Comment:
ah got it.  Well if it's used there can we remove the underscore please?  (I know we haven't been strict on that everywhere in backdraft but it's definitely preferred to indicate it's a "public" API)